deid/cms_deid_init.sql

```diff
@@ -1,10 +1,8 @@
 -- cms_deid_init.sql: Prepare for de-identification of CMS data
--- Copyright (c) 2017 University of Kansas Medical Center

```

Comment:
The norm is to keep the old copyright years when adding new ones, so: `2017-2020`.

not critical.